python/cudf/cudf/tests/test_string.py

```diff
@@ -3423,3 +3423,65 @@ def test_str_join_lists(sr, sep, string_na_rep, sep_na_rep, expected):
         sep=sep, string_na_rep=string_na_rep, sep_na_rep=sep_na_rep
     )
     assert_eq(actual, expected)
+
+
+@pytest.mark.parametrize(
+    "patterns, expected",
+    [
+        (
+            ["a", "s", "g", "i", "o", "r"],
+            [
+                [-1, 0, 5, 3, -1, 2],
+                [-1, -1, -1, -1, 1, -1],
+                [2, 0, -1, -1, -1, 3],
+                [-1, -1, -1, 0, -1, -1],
+            ],
+        ),
+        (
+            ["a", "string", "g", "inn", "o", "r", "sea"],
+            [
+                [-1, 0, 5, -1, -1, 2, -1],
+                [-1, -1, -1, -1, 1, -1, -1],
+                [2, -1, -1, -1, -1, 3, 0],
+                [-1, -1, -1, -1, -1, -1, -1],
+            ],
+        ),
+    ],
+)
+def test_str_find_multiple(patterns, expected):
+    s = cudf.Series(["strings", "to", "search", "in"])
+    t = cudf.Series(patterns)
```

Comment:
Ideally we test the list-like and `Series`-like inputs separately. I don't think we're covering "column-like" inputs at present. Something like
```python
patterns = [
    lambda: cudf.Series(["a", "s", "g", "i", "o", "r"]),
    lambda: ["a", "string", "g", "inn", "o", "r", "sea"],
]
```
and then call `patterns()`. That avoids materializing the GPU Series at collection time, while allowing you to specify both GPU `Series` and `list` inputs.

Reply:
Done 👍